cc/testing/evaluation/algorithms_with_insufficient_noise.h

```diff
@@ -0,0 +1,124 @@
+
+//
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef ALGORITHMS_WITH_INSUFFICIENT_NOISE_H_
+#define ALGORITHMS_WITH_INSUFFICIENT_NOISE_H_
+
+#include "algorithms/count.h"
+#include "algorithms/bounded-sum.h"
+#include "algorithms/bounded-mean.h"
+
+#include <chrono>
+#include <ctime>
+#include <fstream>
+#include <iterator>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+#include <type_traits>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
```

Comment:
Like in the other file, I wonder if these (and possibly other) #includes are needed.